setup.cfg

```
@@ -0,0 +1,57 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for
+ the microservice on aiohttp and prepare the development
+ environment.
+long_description = file: README.md
+long_description_content_type = 'text/markdown'
```

I think that quotes aren't necessary here
```suggestion
long_description_content_type = text/markdown
```